csuntechlab/iSTART
# STR-54 -- FE: Create the admin page with the ability to take in an excel sheet and pass it to the backend

# Description
Allows admin to upload an excel sheet that parses through the file and will send the sheet's contents to the backend.
  
# Testing
1. Upload a file that isnt excel (word, mp3, etc), you should be notified to use an excel file
2. upload the excel file that was sent by me on the iSTART slack channel 
3. to check the parsed file, check the admin component using vue devtools
4. run jest to make sure all tests pass
5. enjoy a cool drink in this pleasant weather
## NOTE: Pressing the submit button will give you a 404 POST  error, this is because the route this is posting to does not exist yet, the route will be created on STR-55.  
## NOTE II, THE SEQUEL: styling is dependent on design's mockup, so it was not included as part of this task

# Installation
Sheets.js's xlsx was used in order to parse through the file. To install, just ``yarn install``
